packages/newspringchurchapp/src/prayer/UserPrayerList/ActionComponent.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ActionSheet from 'react-native-actionsheet';
+import PropTypes from 'prop-types';
+import { Icon, Touchable } from '@apollosproject/ui-kit';
+
+class ActionComponent extends React.Component {
+  handleShowActionSheet = () => {
+    this.ActionSheet.show();
+  };
+
+  render() {
+    const { options } = this.props;
+    return (
+      <>
+        <Touchable onPress={this.handleShowActionSheet}>
+          <Icon name="close" size={18} />
```

Comment:
shouldn't be a hardcoded size here

Reply:
Fixed! Let me know what you think about the red color. At least that more closely symbolizes deletion. I'll keep my eye out for a trash can icon.